impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -384,9 +385,10 @@ std::optional<Entity> GaussianBlurFilterContents::RenderFilter(
     return std::nullopt;
   }
 
-  Vector2 scaled_sigma = (effect_transform.Basis() *
-                          Vector2(ScaleSigma(sigma_x_), ScaleSigma(sigma_y_)))
-                             .Abs();
+  Vector2 scaled_sigma =
+      (effect_transform.Basis() * entity.GetTransform().Basis() *
```

Comment:
Similar comment about direction of the blurs...

Reply:
ack

Reply:
If it isn't an easy fix (it will likely involve modifying the shader to accept a blur vector or something) then it might be good to get the scaling fix in and file the rotation issue as a separate issue...?